flash_examples/video_classification.py

```diff
@@ -34,7 +34,9 @@
 model = VideoClassifier(backbone="x3d_xs", labels=datamodule.labels, pretrained=False)
 
 # 3. Create the trainer and finetune the model
-trainer = flash.Trainer(max_epochs=3, gpus=torch.cuda.device_count(), fast_dev_run=True)
+trainer = flash.Trainer(
+    max_epochs=1, gpus=torch.cuda.device_count(), strategy="ddp" if torch.cuda.device_count() > 1 else None
```

Comment:
isn't ddp default?

Reply:
No, DDP spawn is the default, which doesn't work if your model can't be pickled (in this case the pytorchvideo model can't be)

Reply:
so lets add it also to the comment/note